openquake/hazardlib/gsim/stafford_2022.py

```diff
@@ -0,0 +1,506 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2012-2022 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Author: s.bora@gns.cri.nz /sanjay.singh1986@gmail.com
+Module exports :class:`Stafford2022`
+"""
+import numpy as np
+from scipy import stats
+
+from openquake.baselib.general import CallableDict
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV, SA
+
+CONSTANTS = {"c2": 1.06, "c4": -2.1, "c4a": -0.5, "crb": 50.0,
+             "c8a": 0.2695, "c11": 0.0, "phi6": 300.0, "phi6jp": 800.0}
+
+"""
```

Comment:
I guess the format of these comments does not comply with the one used (that helps to create this documentation https://docs.openquake.org/oq-engine/latest/reference/). I leave this to the IT guys. There are also characters that I am not sure @micheles would be happy with ;)